nextcord/ui/view.py

```diff
@@ -160,6 +160,13 @@ class View:
         Whether or not to automatically defer the component interaction when the callback
         completes without responding to the interaction. Set this to ``False`` if you want to
         handle view interactions outside of the callback.
+    prevent_update: :class:`bool` = True
+        This option only affects persistent views.
```

Comment:
Might want to put the line somewhere towards the bottom of this document

Reply:
I think it's important to let users know at first glance that this is a parameter used for persistent views. If they're not using persistent views, saves some reading

Reply:
I don't think that really changes anything since the table of contents for a class is organized alphabetically.

Reply:
I meant the `This option only affects persistent views.` part specifically, GitHub just made it look like I highlighted the 2 lines

Reply:
I intentionally put it at the top. As aforementioned, I believe it is important to address that it is clear to the user that it should only affect persistent views, and so non-persistent view users wouldn't need to know the rest of the parameter's description before knowing that it isn't necessary for them.